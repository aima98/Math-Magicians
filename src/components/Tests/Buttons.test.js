import renderer from 'react-test-renderer';
import Button from '../Buttons';

function handleClick() {
  return 'message';
}

it('renders correctly', () => {
  const tree = renderer.create(
    <Button value="1" span="span1" bg="bg-color" press={handleClick} />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
