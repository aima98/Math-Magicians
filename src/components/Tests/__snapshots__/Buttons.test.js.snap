// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="button span1bg-color"
  onClick={[Function]}
  type="button"
>
  1
</button>
`;